.gitignore

```diff
@@ -44,4 +44,6 @@ Thumbs.db
 .nx
 
 # env
-.env*
\ No newline at end of file
+.env*
+/packages/sample-app/chromadb
```

Comment:
why?

Reply:
So when we run chromadb locally it creates the files do we need to commit those? It should be local right?